### PR TITLE
Update OutputIdsResponse

### PR DESCRIPTION
**Pull request by Thoralf-M**
# Description of change

Update OutputIdsResponse

## Type of change

- Bug fix (a non-breaking change which fixes an issue)

## How the change has been tested

Running 07_get_random_basic_outputs.rs

### CODE REVIEW

File: sdk/src/types/api/plugins/indexer.rs [hunk 1]
```diff
@@ -17,8 +17,11 @@ use crate::types::block::output::OutputId;
     serde(rename_all = "camelCase")
 )]
 pub struct OutputIdsResponse {
-    /// The ledger index at which the outputs were collected
-    pub ledger_index: u32,
+    /// The committed slot at which these outputs where available at.
```
Comment (thibault-martinez) [inline sdk/src/types/api/plugins/indexer.rs]:
```suggestion
    /// The committed slot at which these outputs were available at.
```

File: bindings/python/iota_sdk/client/_node_indexer_api.py [hunk 2]
```diff
@@ -9,17 +9,21 @@
...
 
 
+@json
+@dataclass
 class OutputIdsResponse:
     """Response type for output IDs.
 
     Attributes:
-        ledger_index: The ledger index for which the response is valid.
+        committed_slot: The committed slot at which these outputs where available at.
```
Comment (thibault-martinez) [inline bindings/python/iota_sdk/client/_node_indexer_api.py]:
```suggestion
        committed_slot: The committed slot at which these outputs were available at.
```

File: bindings/nodejs/lib/types/models/api/plugins/indexer/output-response.ts [hunk 3]
```diff
@@ -9,11 +9,11 @@ import type { HexEncodedString } from '../../../../utils/hex-encoding';
  */
 export interface IOutputsResponse {
     /**
-     * The ledger index at which these outputs where available at.
+     * The committed slot at which these outputs where available at.
```
Comment (thibault-martinez) [inline bindings/nodejs/lib/types/models/api/plugins/indexer/output-response.ts]:
```suggestion
     * The committed slot at which these outputs were available at.
```

File: sdk/src/types/api/plugins/indexer.rs (hunk 1 above)
Comment (Thoralf-M) [reply sdk/src/types/api/plugins/indexer.rs]: Please also comment on the TIP :trollface:

Comment (thibault-martinez) [reply sdk/src/types/api/plugins/indexer.rs]: Done

File: bindings/nodejs/lib/types/models/api/output-metadata-response.ts [hunk 4]
```diff
@@ -45,5 +45,5 @@ export interface IOutputMetadataResponse {
     /**
      * The ledger index at which these output was available at.
      */
-    ledgerIndex: number;
+    committedSlot: number;
```
Comment (Thoralf-M) [inline bindings/nodejs/lib/types/models/api/output-metadata-response.ts]:
```suggestion
    ledgerIndex: number;
```

**Review by thibault-martinez — APPROVED** [review]

**Review by kwek20 — APPROVED** [review]

**Review by Alex6323 — APPROVED** [review]

**Pull request merged**